### DISCUSSION TITLE
Anything New??

### DISCUSSION THREAD

**Discussion by doogo809** [NexTre-dev/alphaver#103] [General]
Has there been anything new with 1.0.16 yet?

**Comment by mbccompanyx** [comment]
If there was anything new we would have made an update, frankly i'm not surprised we're going through longer periods of radio silence between drops

**Comment by Vladg24YT** [comment]
There was an event, but it's still debated whether it was a gamejacking attempt or an actual 14th sighting.

**Comment by mbccompanyx** [comment]
I wouldn't be surprised if was some game jacking, if we still consider retro gaming now message about sightings real or him being a game jacker
